WordOfTheDay.PerformanceCheck/WordOfTheDay.PerformanceCheck/PerformanceCheck.cs

```diff
@@ -21,78 +20,4 @@ static async Task Main()
             await request.MinMaxAvgTime();
```

Comment:
When I read the name of this method, I don't understand what it does. Can you rename it somehow?